frontend/src/components/utils/autoRefreshHandler.tsx

```diff
@@ -69,17 +69,29 @@ export default (
     };
 
     fetchInfo = (count: number) => {
+      let newState = this.state;
```

Comment:
You are fortunate enough here that you re-assigned this newState later... You should never modify `self.state` as this will hit you back in a very unexpected way. You don't need the old state for `setState`. I will update the code.